stdlib/2/commands.pyi

```diff
@@ -3,3 +3,6 @@ from typing import Tuple
 def getstatus(file: str) -> str: ...
 def getoutput(cmd: str) -> str: ...
 def getstatusoutput(cmd: str) -> Tuple[int, str]: ...
+
+def mk2arg(head: str, x: str) -> str: ...
+def mkarg(x: str) -> str: ...
```

Comment:
This should be AnyStr -> AnyStr I think.

Reply:
I presume that whatever the answer is for `mkarg()`, should also be applied to the 3 get* functions in this module.

Reply:
No, for them the argument type doesn't affect the return type. I think their argument types should be Text (which means in Python 2 they accept both `str` and `unicode`) and their return types can stay as they are.